src/components/Class/ClassCard/styles.scss

```diff
@@ -0,0 +1,46 @@
+.class-cards {
```

Comment:
It should be the singular name. `class-card` could be much better? what do you think? 🤔

Reply:
done